src/functions.php

```diff
@@ -1055,3 +1055,29 @@ function guard_min_version(string $minVersion): void
         throw new MinimumVersionRequirementNotMetException($minVersion, $currentVersion);
     }
 }
+
+/**
+ * @param string|non-empty-array<string> $url
+ */
+function open(string|array $url): void
```

Comment:
Did you consider doing
```suggestion
function open(string ...$url): void
```

Reply:
I like it !